tests/testing_data/integration_answers.py

```diff
@@ -493,6 +493,73 @@
             "infer_metric": 0.9316383600234985,
         },
     },
+    {
+        # test answers for PyTorch 21.10 v1
+        "integration_segmentation_3d": {
+            "losses": [
+                0.5373653262853623,
+                0.46776085495948794,
+                0.4422474503517151,
+                0.43667820692062376,
+                0.42639826238155365,
+                0.4158218264579773,
+            ]
+        }
+    },
+    {  # test answers for PyTorch 21.10 mixed readers
```

Comment:
May I know what "mixed readers" mean here?

Thanks.

Reply:
yes that means in this for loop of testing different combinations, not all of them failed, this answer matches some of them...

https://github.com/Project-MONAI/MONAI/blob/5657b8f2d297e2d2b09017cf9f57dad2a72186f4/tests/test_integration_segmentation_3d.py#L291-L292